%-------------------------
% Resume in Latex
% Author : Aras Gungore
% License : MIT
%------------------------

\documentclass[letterpaper,11pt]{article}

\usepackage{latexsym}
\usepackage[empty]{fullpage}
\usepackage{titlesec}
\usepackage{marvosym}
\usepackage[usenames,dvipsnames]{color}
\usepackage{verbatim}
\usepackage{enumitem}
\usepackage{hyperref}
\usepackage{fancyhdr}
\usepackage[english]{babel}
\usepackage{tabularx}
\usepackage{hyphenat}
\usepackage{fontawesome}
\input{glyphtounicode}
\usepackage{xcolor}

% Define the blue color for hyperlinks
\definecolor{linkcolor}{rgb}{0,0,0.5}

% Configure hyperlink setup
\hypersetup{
    colorlinks=true,
    linkcolor=linkcolor,
    urlcolor=linkcolor,
    citecolor=linkcolor
}


%---------- FONT OPTIONS ----------
% sans-serif
% \usepackage[sfdefault]{FiraSans}
% \usepackage[sfdefault]{roboto}
% \usepackage[sfdefault]{noto-sans}
% \usepackage[default]{sourcesanspro}

% serif
% \usepackage{CormorantGaramond}
% \usepackage{charter}


\pagestyle{fancy}
\fancyhf{} % clear all header and footer fields
\fancyfoot{}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{0pt}

% Adjust margins
\addtolength{\oddsidemargin}{-0.5in}
\addtolength{\evensidemargin}{-0.5in}
\addtolength{\textwidth}{1in}
\addtolength{\topmargin}{-.5in}
\addtolength{\textheight}{1.0in}

\urlstyle{same}

\raggedbottom
\raggedright
\setlength{\tabcolsep}{0in}

% Sections formatting
\titleformat{\section}{
  \vspace{-4pt}\scshape\raggedright\large
}{}{0em}{}[\color{black}\titlerule \vspace{-5pt}]

% Ensure that generate pdf is machine readable/ATS parsable
\pdfgentounicode=1

%-------------------------
% Custom commands

\newcommand{\resumeItem}[1]{
  \item\small{
    {#1 \vspace{-2pt}}
  }
}


\newcommand{\resumeSubheading}[4]{
  \vspace{-2pt}\item
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{#1} & #2 \\
      \textit{\small#3} & \textit{\small #4} \\
    \end{tabular*}\vspace{-7pt}
}


\newcommand{\resumeSubSubheading}[2]{
    \vspace{-2pt}\item
    \begin{tabular*}{0.97\textwidth}{l@{\extracolsep{\fill}}r}
      \textit{\small#1} & \textit{\small #2} \\
    \end{tabular*}\vspace{-7pt}
}


\newcommand{\resumeEducationHeading}[6]{
  \vspace{-2pt}\item
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{#1} & #2 \\
      \textit{\small#3} & \textit{\small #4} \\
      \textit{\small#5} & \textit{\small #6} \\
    \end{tabular*}\vspace{-5pt}
}


\newcommand{\resumeProjectHeading}[2]{
    \vspace{-2pt}\item
    \begin{tabular*}{0.97\textwidth}{l@{\extracolsep{\fill}}r}
      \small#1 & #2 \\
    \end{tabular*}\vspace{-7pt}
}


\newcommand{\resumeOrganizationHeading}[4]{
  \vspace{-2pt}\item
    \begin{tabular*}{0.97\textwidth}[t]{l@{\extracolsep{\fill}}r}
      \textbf{#1} & \textit{\small #2} \\
      \textit{\small#3}
    \end{tabular*}\vspace{-7pt}
}

\newcommand{\resumeSubItem}[1]{\resumeItem{#1}\vspace{-4pt}}

\renewcommand\labelitemii{$\vcenter{\hbox{\tiny$\bullet$}}$}

\newcommand{\resumeSubHeadingListStart}{\begin{itemize}[leftmargin=0.15in, label={}]}
\newcommand{\resumeSubHeadingListEnd}{\end{itemize}}
\newcommand{\resumeItemListStart}{\begin{itemize}}
\newcommand{\resumeItemListEnd}{\end{itemize}\vspace{-5pt}}

%-------------------------------------------
%%%%%%  RESUME STARTS HERE  %%%%%%%%%%%%%%%%%%%%%%%%%%%%


\begin{document}

%---------- HEADING ----------

\begin{center}
    {\Huge\bfseries Phil Palmer} \\ \vspace{3pt}
    % mail to
    \href{mailto:pp502@cam.ac.uk}{\color{teal}pp502@cam.ac.uk} \\
    % \small born 26 August 1997 $|$ British $|$ 
    \href{https://philpalmer.github.io/blog/}{\color{teal}Website} $|$ \href{https://github.com/PhilPalmer}{\color{teal}GitHub} $|$ \href{https://uk.linkedin.com/in/phil-palmer}{\color{teal}LinkedIn} $|$ \href{https://scholar.google.com/citations?user=VMrmNJcAAAAJ}{\color{teal}Google Scholar}  $|$  +44 7531 445 903 \\
\end{center}



%----------- EDUCATION -----------

\section{\textbf{Education}}
% \vspace{3pt}
\begin{tabularx}{\linewidth}{@{}p{2cm}@{\hspace{5pt}}|@{\hspace{5pt}}X@{}}
    2020 -- now & 
    \textbf{PhD Biology $|$ University of Cambridge} \\
    & \begin{minipage}[t]{\linewidth}
        \begin{itemize}[noitemsep]
            \item PhD in the \href{https://www.lvz.vet.cam.ac.uk/}{\color{teal}Lab of Viral Zoonotics} supervised by Prof. \href{https://www.infectiousdisease.cam.ac.uk/directory/jlh66\%40cam.ac.uk}{\color{teal}Jonathan Heeney}.
            \item Researching computational methods to design broad-spectrum vaccines and antibodies against antigenically diverse pathogens.
            % \item Developed a novel computational method to design broad-spectrum vaccines and applied it to both coronaviruses and influenza viruses.
        \end{itemize}
    \end{minipage} \\
    2015 -- 2018 &
    \textbf{BSc Biology (First Class Honours) $|$ University of Southampton} \\
    & \begin{minipage}[t]{\linewidth}
        \begin{itemize}[noitemsep]
            \item Focused on bioinformatics, with my final year project supervised by Dr. \href{https://www.southampton.ac.uk/people/5wycg9/doctor-jane-gibson}{\color{teal}Jane Gibson}.
            \item Scored 86\% in my thesis project processing over 1TB of data to identify genetic variants in bladder cancer, using machine learning classifiers for accurate sample differentiation.
        \end{itemize}
    \end{minipage}
\end{tabularx}



%----------- RESEARCH -----------

\section{\textbf{Research Internships}}
% \vspace{3pt}
\begin{tabularx}{\linewidth}{@{}p{2cm}@{\hspace{5pt}}|@{\hspace{5pt}}X@{}}
    Oct. 2022 -- & 
    \textbf{Visiting Researcher $|$ Massachusetts Institute of Technology (MIT)} \\
    Jan. 2023 & \begin{minipage}[t]{\linewidth}
        \begin{itemize}[noitemsep]
            \item Evaluated and improved a general‐purpose metagenomics pipeline for the \href{https://naobservatory.org/}{\color{teal}Nucleic Acid Observatory}, a project that aims to build a reliable early detection system for catastrophic biothreats.
            \item Supervised by Prof. \href{https://www.media.mit.edu/people/esvelt/overview/}{\color{teal}Kevin Esvelt} and Dr. Mike McLaren.
        \end{itemize}
    \end{minipage}
\end{tabularx}


%----------- WORK -----------

\section{\textbf{Work Experience}}
\vspace{3pt}
\begin{tabularx}{\linewidth}{@{}p{2cm}@{\hspace{5pt}}|@{\hspace{5pt}}X@{}}
  2018 -- 2020 & 
  \textbf{Bioinformatician $|$ Lifebit Biotech} \\
  & \begin{minipage}[t]{\linewidth}
      \begin{itemize}[noitemsep]
          \item Was instrumental in the end‐to‐end delivery of numerous key projects, including the \href{https://www.genomicsengland.co.uk/news/research-environment-covid-19-lifebit-aws}{\color{teal}Genomics England Research Platform} for COVID‐19 response.
          \item My role included everything from gathering technical requirements to building bioinformatics pipelines and providing professional technical customer support.
          \item Contributed to a robust imputation pipeline used by direct-to-consumer genetics companies like Nebula Genomics and Sano Genetics for analysing thousands of their customers' data.
      \end{itemize}
  \end{minipage}
\end{tabularx}



%----------- SERVICE -----------

\section{\textbf{Service}}
\begin{tabularx}{\linewidth}{@{}p{2cm}@{\hspace{5pt}}|@{\hspace{5pt}}X@{}}
    2020 -- now & 
    \textbf{Co-founder of the Cambridge Biosecurity Hub} \\
    & Scaled the society from 0 to 30+ members and helped organise weekly reading groups and 7-week \href{https://lacy-trouser-5c8.notion.site/Cambridge-Biosecurity-Hub-Research-Projects-a0801773e5c54517b2e33e70cc572167?pvs=4}{\color{teal}projects} on biosecurity topics. 
\end{tabularx}


%----------- AWARDS -----------

\section{\textbf{Fellowships and Awards}}
\begin{tabularx}{\linewidth}{@{}p{2cm}@{\hspace{5pt}}|@{\hspace{5pt}}X@{}}
    2020 & 
    \textbf{Doctor of Philosophy Scholarship $|$ Open Philanthropy} \\
    & For three years of PhD studies at the University of Cambridge. \\
    2019 & 
    \textbf{Winner of the world’s first clean meat hackathon $|$ HigherSteaks} \\
    & Helped develop and pitch our innovative idea for the use of machine learning to optimise the recycling of cell media, a key factor in reducing costs in cellular agriculture. \\
\end{tabularx}


%----------- SKILLS -----------

\section{\textbf{Programming Skills}}
\begin{tabularx}{\linewidth}{@{}p{2cm}@{\hspace{5pt}}|@{\hspace{5pt}}X@{}}
    \texttt{python} & \textit{Highly competent}: data analysis/visualisation, machine learning, algorithm implementation \\
    \texttt{nextflow} & \textit{Competent}: workflow management, pipeline development \\
    \texttt{R} & \textit{Competent}: data analysis/visualisation \\
    System & \textit{Competent}: Git, Linux, Docker, working with HPC clusters, AWS \\
    Web & \textit{Competent}: HTML5, CSS3, Javascript
\end{tabularx}


%----------- TALKS -----------

\section{\textbf{Talks}}
\begin{tabularx}{\linewidth}{@{}p{2cm}@{\hspace{5pt}}|@{\hspace{5pt}}X@{}}
    2019 & 
    \textbf{Workshop on Nextflow and Containers} \\
    & Prepared and taught a day-long \href{https://github.com/lifebit-ai/jax-tutorial}{\color{teal}workshop} to 45 participants at the Jackson Laboratory, CT, USA. \\
\end{tabularx}


%----------- PUBLICATIONS -----------

\section{\textbf{Publications}}

\subsection*{\textbf{Journal Articles}}
\begin{tabularx}{\linewidth}{@{}p{2cm}@{\hspace{5pt}}|@{\hspace{5pt}}X@{}}
    2022 & 
    \textit{\href{https://doi.org/10.1016/j.celrep.2022.111704}{\color{teal}Myc regulates a pan-cancer network of co-expressed oncogenic splicing factors}} $|$ Cell Reports \\
    & Urbanski L, Brugiolo M, Park S, Angarola BL, Leclair NK, Yurieva M, \textbf{Palmer P}, Sangram S, Olga A. \\
    
    2022 & 
    \textit{\href{https://doi.org/10.3389/fimmu.2022.773982}{\color{teal}Neutralisation Hierarchy of SARS-CoV-2 Variants of Concern Using Standardised, Quantitative Neutralisation Assays Reveals a Correlation With Disease Severity; Towards Deciphering Protective Antibody Thresholds}} $|$ Frontiers in Immunology \\
    & Cantoni D, Mayora-Neto M, Nadesalingam A, Wells DA, Carnell GW, Ohlendorf L, Ferrari M, \textbf{Palmer P}, Chan AC, Smith P, Bentley EM, Einhauser S, Wagner R, Page M, Raddi G, Baxendale H, Castillo-Olivares J, Heeney J, Temperton N. \\
    
    2022 & 
    \textit{\href{https://doi.org/10.3389/fimmu.2021.681636}{\color{teal}AutoPlate: Rapid Dose-Response Curve Analysis for Biological Assays}} $|$ Frontiers in Immunology \\
    & \textbf{Palmer P}, Del Rosario JMM, da Costa KA, Carnell GW, Huang CQ, Heeney JL, Temperton NJ. \\
    
    2021 & 
    \textit{\href{https://doi.org/10.3389/fimmu.2021.748291}{\color{teal}Analysis of serological biomarkers of SARS-COV-2 infection in convalescent samples from severe, moderate and mild covid-19 cases}} $|$ Frontiers in Immunology \\
    & Castillo-Olivares J, Wells DA, Ferrari M, Chan AC, Smith P, Nadesalingam A, Paloniemi M, Carnell GW, Ohlendorf L, Cantoni D, Mayora-Neto M, \textbf{Palmer P}, Tonks P, Temperton NJ, Peterhoff D, Neckermann P, Wagner R, Doffinger R, Kempster S, Otter AD, Semper A, Brooks T, Albecka A, James LC, Page M, Schwaeble W, Baxendale H, Heeney JL. \\
\end{tabularx}

\subsection*{\textbf{Manuscripts in Preparation}}
\begin{tabularx}{\linewidth}{@{}p{2cm}@{\hspace{5pt}}|@{\hspace{5pt}}X@{}}
    2024 & 
    \textit{A general purpose method to design broad-spectrum T-cell vaccines applied to coronavirus and influenza A antigens} $|$ In preparation \\
    & \textbf{Palmer P}, Vishwanath S, Fedosyuk S, Carnell GW, Holbrook J, Parthasarathy S, Heeney JL. \\

    2024 & 
    \textit{A benchmark of deep learning methods for antibody-antigen affinity prediction} $|$ In preparation \\
    & \textbf{Palmer P}, Mathis SV, Del Rosario JMM, Orbegoso CC, Heeney JL, P Liò. \\
\end{tabularx}



%----------- Software -----------

\section{\textbf{Software}}
\vspace{3pt}
\begin{tabularx}{\linewidth}{@{}p{2cm}@{\hspace{5pt}}|@{\hspace{5pt}}X@{}}
    2023 & 
    \textbf{Contributor $|$ \href{https://github.com/nf-core/mag}{\color{teal}nf-core/mag}} (161 stars) \\
    & A Nextflow pipeline for the assembly and binning of metagenomes. I contributed to the development of the pipeline by adding the geNomad tool for viral classification. \\
    2021 &
    \textbf{Primary Developer $|$ \href{https://github.com/PhilPalmer/AutoPlate}{\color{teal}PhilPalmer/AutoPlate}} (4 stars) \\
    & An RShiny app to automate the analysis of 96-well plates such as the fitting of dose-response curves. I developed the app during my PhD at the University of Cambridge to help with the analysis of assays for SARS-CoV-2 and influenza viruses. \\
    2020 &
    \textbf{Primary Developer $|$ \href{https://github.com/TheJacksonLaboratory/splicing-pipelines-nf}{\color{teal}TheJacksonLaboratory/splicing-pipelines-nf}} (15 stars) \\
    & A Nextflow pipeline for the detection of alternative splicing events in RNA-seq data. I developed the pipeline during collaborative research with the Olga Anczuków lab, at the Jackson Laboratory, during my role as a bioinformatician at Lifebit. \\
    2020 &
    \textbf{Contributor $|$ \href{https://github.com/apriha/snps}{\color{teal}apriha/snps}} (82 stars) \\
    & A Python package for the analysis of single nucleotide polymorphisms (SNPs). I contributed to the development of the package by identifying and fixing a bug in reading 23andMe files. \\
    2018 &
    \textbf{Primary Developer $|$ \href{https://github.com/nf-core/DeepVariant}{\color{teal}nf-core/DeepVariant}} (40 stars) \\
    & A Nextflow pipeline for running Google's DeepVariant which uses deep neural networks to call genetic variants from next-generation DNA sequencing data. I developed this as one of the first pipelines for the open-source nf-core community during my role as a bioinformatician at Lifebit. \\    
\end{tabularx}



%----------- RELEVANT COURSEWORK -----------

% \section{Relevant Coursework}
  % \vspace{2pt}
  % \resumeSubHeadingListStart
    % \small{\item{
        % \textbf{Major coursework:}{ Materials Science, Electrical Circuits I-II, Digital System Design, Numerical Methods, Probability Theory, Electronics I-II, Signals and Systems, Electromagnetic Field Theory, Energy Conversion, System Dynamics and Control, Communication Engineering, Pattern Recognition, Introduction to Digital Signal Processing, Introduction to Digital Communications, Introduction to Database Systems, Introduction to Image Processing, Machine Vision} \\ \vspace{3pt}
        
        % \textbf{Minor coursework:}{ Discrete Computational Structures, Introduction to Object-Oriented Programming, Data Structures and Algorithms, Computer Organization, Fundamentals of Software Engineering}
    % }}
  % \resumeSubHeadingListEnd



%----------- CERTIFICATES -----------

% \section{Certificates}
  % \resumeSubHeadingListStart
    
    % \resumeOrganizationHeading
      % {Procter \& Gamble VIA Certificate Program}{Feb 2022}{Business Skills, Data and Digital Skills, Project Management and Personal Productivity}
    
  % \resumeSubHeadingListEnd



%----------- ORGANIZATIONS -----------

% \section{Organizations}
  % \resumeSubHeadingListStart
    
    % \resumeOrganizationHeading
      % {Institute of Electrical and Electronics Engineers (IEEE)}{Feb 2022 -- Present}{Student Member}
    
  % \resumeSubHeadingListEnd



%----------- HOBBIES -----------

% \section{Hobbies}
  % \resumeSubHeadingListStart
    % \small{\item{Basketball, Swimming, Fitness, Eight-ball, Horology}}
  % \resumeSubHeadingListEnd



%----------- REFERENCES -----------

% \section{References}
  % \vspace{2pt}
  % \resumeSubHeadingListStart
    % \item{References available upon request.}
  % \resumeSubHeadingListEnd



%-------------------------------------------
\end{document}
